### PR TITLE
configs for EC2 rpcs

### PR DESCRIPTION
**Pull request by wanwiset25**

### CODE REVIEW

File: devnet/docker-compose.yml [hunk 1]
```diff
@@ -1,17 +1,10 @@
 version: "3.4"
 
 services:
-  xinfinnetwork1:
-    build:
-        context: .
-        dockerfile: ./Dockerfile
+  xinfinnetwork:
+    image: ${RPC_IMAGE}
```
Comment (liam-lai) [inline devnet/docker-compose.yml:5]: Can you create another compose file for all the network, current docker compose file is widely used by other members, and if we change the format, we need to manage all the conversation and way of deployment things from their end.

File: mainnet/docker-up.sh [hunk 2]
```diff
@@ -7,4 +7,4 @@ if [[ $? != 0 ]]; then
     alias docker-compose='docker compose'
 fi
 
-docker-compose -f docker-compose.yml up -d
+HOSTIP=$(curl https://checkip.amazonaws.com) docker-compose -f docker-compose.yml up -d --build --force-recreate
```
Comment (liam-lai) [inline mainnet/docker-up.sh:10]: HOSTIP=$(curl https://checkip.amazonaws.com) is not required

File: testnet/docker-up.sh [hunk 3]
```diff
@@ -7,4 +7,4 @@ if [[ $? != 0 ]]; then
     alias docker-compose='docker compose'
 fi
 
-docker-compose -f docker-compose.yml up -d --build --force-recreate
+HOSTIP=$(curl https://checkip.amazonaws.com) docker-compose -f docker-compose.yml up -d --build --force-recreate
```
Comment (liam-lai) [inline testnet/docker-up.sh:10]: HOSTIP=$(curl https://checkip.amazonaws.com/) is not required

**Comment by wanwiset25** [comment]
@liam-lai Please check again

**Review by liam-icheng-lai — APPROVED** [review]

**Review by liam-lai — APPROVED** [review]

**Pull request merged**